.env.example

```
@@ -63,3 +63,7 @@ SUPERMEMORY_API_KEY=sm_...
 SUPERMEMORY_ENABLED=true
 # Optional: MISTRAL_OCR_ENDPOINT=https://api.mistral.ai/v1/ocr
 # Optional: MISTRAL_OCR_MODEL=mistral-ocr-latest
+# Zero Sync
+NEXT_PUBLIC_ZERO_SERVER=http://localhost:4848
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ZERO_SERVER key should go before the SUPERMEMORY_ENABLED key

(UnorderedKey)